CI.md

```diff
@@ -125,6 +125,13 @@ Service Account shall have following permissions ([IAM roles](https://cloud.goog
 | Java Wordcount Direct Runner | Runs Java WordCount example with Direct Runner.                                               | Yes              | Yes                   | Yes           | -                        |
 | Java Wordcount Dataflow      | Runs Java WordCount example with DataFlow Runner.                                             | -                | Yes                   | Yes           | Yes                      |
 
+### Release Preparation and Validation Workflows
+#### Verify Release Build - [verify_release_build.yml](.github/workflows/verify_release_build.yml)
+| Job                          | Description                                                                                   | Pull Request Run | Direct Push/Merge Run | Scheduled Run | Requires GCP Credentials |
+|------------------------------|-----------------------------------------------------------------------------------------------|------------------|-----------------------|---------------|--------------------------|
+| Verify Release Build         | Verifies full life cycle of Gradle Build and all PostCommit/PreCommit tests against Release Branch on Jenkins.                   | No               | No                    | No            | No                       |
```

Comment:
Nit: "on Jenkins" documentation outdated

Reply:
Could you please add more details on what should I do in this line? Thank you!

Reply:
```suggestion
| Verify Release Build         | Verifies full life cycle of Gradle Build and all PostCommit/PreCommit tests against Release Branch on CI.                   | No               | No                    | No            | No                       |
```

Could we just make it generic?